bin/content-push

```diff
@@ -73,7 +86,8 @@ def commit_changes(name)
 
   branchname = GitUtils.current_branch
   exit 1 unless system("git add -A #{CONTENT_PATHS}")
-  exit 1 unless system("git commit #{CONTENT_PATHS} -m '#{branchname} content changes (-#{name})'")
+  ignore_levelbuilder_omissions
+  exit 1 unless system("git commit -m '#{branchname} content changes (-#{name})'")
```

Comment:
just so that I understand, are you changing the behavior on this line or just removing an unnecessary but harmless `#{CONTENT_PATHS}`? the new syntax looks good, but I'm not sure I understand what the old syntax would do.

Reply:
The old command ignored the files added by the `git add` on the previous line, and committed only the specified files (of course both commands used the same set of files, so it didn't matter). The new one commits everything that has been added, and ignores anything un-added by `ignore_levelbuilder_omissions`.

This change is no longer necessary now that I actually revert the changes rather than just resetting the file. I still think it's clearer this way though, since it's closer to the way most people use `git add` and `git commit`.